crates/orderbook/openapi.yml

```diff
@@ -593,6 +598,31 @@ components:
       required:
         - expiration
         - amount
+    OnchainOrderData:
```

Comment:
Why is this information is in `OnchainOrderData` and not `EthflowData`? I would think, each smart order will have their own logic so not necessarily there will always be this info, right? 

for example, a smart order that is just a pool of people trading? (you cannot identify a single user, there will be a poolId or something else)

Reply:
>Why is this information is in OnchainOrderData and not EthflowData? I would think, each smart order will have their own logic so not necessarily there will always be this info, right?

No, we devised a generic order placement procedure for onchain orders. And all data part of the generic concept ended up in the OnchainOrderData struct. Components that are specific to ethflow orders end up in their own struct.
It looks weird at first look, but then it makes more sense, once one knows that we have this structure prepared for future on-chain order placements.